src/data/sheetLoadingHelpers.js

```diff
@@ -45,34 +45,19 @@ export function callSheets(selected = "", filterType = "") {
         category.forEach(cat => categories[cat] = cat.trim());
         for (let tag of project.tags) { tags[tag] = "" };
       }
-      const categoryList = [...(new Set(Object.values(categories)))];
-
-      var my_json = JSON.stringify(data);
-
-      if (selected.length > 0 && filterType == "category") {
-        filter_criteria_list = update_criteria(selected, filter_criteria_list);
-      }
-
-      filtered_json = filter_criteria_list.length <= 0 ? data : find_in_object(JSON.parse(my_json), { categoryautosortscript: filter_criteria_list });
 
-      if (selected.length > 0 && filterType == "name") {
-        filtered_json = filtered_json.filter(function (i) {
-          return i.name.toLowerCase().match(selected.toLowerCase());
-        });
-      }
-
-      filtered_json = filtered_json.filter(function (org) { return org.truefalsevetting === 'TRUE' });
+      const categoryList = [...(new Set(Object.values(categories)))];
 
-      filtered_json.forEach(obj => { obj.isMarkerOpen = false; });
+        data = data.filter(function (org) { return org.truefalsevetting === 'TRUE' });
 
-      //console.log(filtered_json)
+        data.forEach(obj => { obj.isMarkerOpen = false; });
 
 
       //This creates a hash table based for the lat and long of each loction.
       //This allows us to group all organizations at the same location together. 
       var locationAddressHashTable = {};
 
-      Object.entries(filtered_json).forEach(([index,  org]) =>{
+      Object.entries(data).forEach(([index,  org]) =>{
 
       if(org.coordinates){
```

Comment:
What is the value of `org.coordinates` ? We might want to have a better presence check here to avoid anything weird coming in. For instance, if `org.coordinates` was a string, it'd be true, if it was an empty array it'd be true, etc. If `org.coordinates` is an object, ie: `{x: 10, y: 74}` then we might be able to check if those values exist:

```
const { xCoordinate, yCoordinate } = org.coordinates;

if (xCoordinate && yCoordinate) {
  ...
}
```

Something along those lines. This will be sure that we aren't passing into the true section by accident, or trying to `createMarkerId` on incorrect data.

Reply:
This is used to create the locationAddressHashTable which is grouping the objects based on the coordinates. This is currently being refactored.